lib/pages/chat/reply_display.dart

```diff
@@ -23,33 +23,36 @@ class ReplyDisplay extends StatelessWidget {
           : 0,
       clipBehavior: Clip.hardEdge,
       decoration: const BoxDecoration(),
-      child: Row(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: <Widget>[
-          const SizedBox(
-            width: 56,
-          ),
-          Expanded(
-            child: controller.replyEvent != null
-                ? ReplyContent(
-                    controller.replyEvent!,
-                    timeline: controller.timeline!,
-                    chatController: controller,
-                  )
-                : _EditContent(
-                    controller.editEvent?.getDisplayEvent(controller.timeline!),
-                  ),
-          ),
-          IconButton(
-            padding: EdgeInsets.zero,
-            tooltip: L10n.of(context)!.close,
-            icon: Icon(
-              Icons.close,
-              color: Theme.of(context).colorScheme.primary,
+      child: SingleChildScrollView(
```

Comment:
IMO: If you use `SingleChildScrollView` we can scroll now? Should we not allow scrolling here?

Reply:
You're right, as discussed this morning we'll only show one line now. I've updated the html package here : https://github.com/linagora/flutter_matrix_html/pull/10 please check it out